x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -402,3 +405,8 @@ export interface AlertsTableProps {
   useFetchAlertsData: () => FetchAlertData;
   'data-test-subj': string;
 }
+
+export interface AlertsTableConfigurationRegistry {
```

Comment:
Do we need to export it publicly?

Reply:
Yes. This will be used by the other plugins when they register their configuration.